javascript/game.js

```diff
@@ -1,117 +1,69 @@
-var orange = Math.floor(Math.random() * 19) + 1;
-var banana = Math.floor(Math.random() * 19) + 1;
-var berry = Math.floor(Math.random() * 19) + 1;
-
-// console log for clarity
-console.log(cherry, orange, banana, berry);
-
-
-
-// var total is will be the value of var totalGuess added together
-var total = 0;
-
-win = 0;
-loss = 0;
-
-function winReset(){
-
-	gameWin = Math.floor(Math.random() * 120) + 19;
-
-	// display gameWin in #gameNumber div
-	$("#gameNumber").html(gameWin);
-
-	// create random numbers for fruit
-
-	var cherry = Math.floor(Math.random() * 19) + 1;
-	var orange = Math.floor(Math.random() * 19) + 1;
-	var banana = Math.floor(Math.random() * 19) + 1;
-	var berry = Math.floor(Math.random() * 19) + 1;
-
-	win++;
-	alert("you win");
-	total = 0;
-	$("#playerScore").html(total);
-	$("#wins").html("Wins: " + win);
-	console.log(cherry, orange, banana, berry);
-
-
-};
-
-function loseReset(){
-	gameWin = Math.floor(Math.random() * 120) + 19;
-
-	// display gameWin in #gameNumber div
-	$("#gameNumber").html(gameWin);
-
-	// create random numbers for fruit
-
-	var cherry = Math.floor(Math.random() * 19) + 1;
-	var orange = Math.floor(Math.random() * 19) + 1;
-	var banana = Math.floor(Math.random() * 19) + 1;
-	var berry = Math.floor(Math.random() * 19) + 1;
-
-	
-	alert("you lose");
-	loss++;
-	total = 0;
-	$("#playerScore").html(total);
-	$("#loss").html("Losses: " + loss);
-	console.log(cherry, orange, banana, berry);
+		total = 0;
+		$("#playerScore").html(total);
+	};
 
-}
+	function generateRandomNumber (min, max) {
+		max -= min
+		return Math.floor(Math.random() * max) + min;
+	};
 
-function playGame(){
+	function winReset () {
+		win ++;
+		alert("you win");
+		$("#wins").html("Wins: " + win);
+		resetGameNumbers();
+	};
 
-	$("#playerScore").html(total);
-	$("#wins").html("Wins: " + win);
-	$("#loss").html("Losses: " + win);
+	function loseReset () {
+		loss ++;
+		alert("you lose");
+		$("#loss").html("Losses: " + loss);
+		resetGameNumbers();
+	};
 
-	// when cherry is clicked
-	$(".fruitimg").on("click", function(event){
+	function playGame () {
 
-		if(event.currentTarget.attributes.id.nodeValue === "cherry"){
-	// adds totoal with value of fruit
-		total = total + cherry;
 		$("#playerScore").html(total);
-		// show what our totoal is
-		console.log(total);
-		console.log();
-		}
+		$("#wins").html("Wins: " + win);
+		$("#loss").html("Losses: " + win);
 
-		if(event.currentTarget.attributes.id.nodeValue === "orange"){
-			total = total + orange;
-			$("#playerScore").html(total);
-		}
-		if(event.currentTarget.attributes.id.nodeValue === "banana"){
-			total = total + banana;
-			$("#playerScore").html(total);
-		}
-		if(event.currentTarget.attributes.id.nodeValue === "berry"){
-			total = total + berry;
-			$("#playerScore").html(total);
-		}
+		$(".fruitimg").on("click", function(event){
 
-		if(gameWin < total) {
-			loseReset();
+			total += $(this).data('fruit-val')
```

Comment:
Another plus of storing the random value with the jQuery `.data` method is that you don't have to care about which fruit has been clicked on since the value is stored under the same key for each of them.